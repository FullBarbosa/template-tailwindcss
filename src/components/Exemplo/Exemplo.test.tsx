import { render, screen } from '@testing-library/react'

import { Exemplo } from '.'

describe('<Exemplo />', () => {
  it('should render the heading', () => {
    const { container } = render(<Exemplo />)

    expect(
      screen.getByRole('heading', { name: /Exemplo/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
