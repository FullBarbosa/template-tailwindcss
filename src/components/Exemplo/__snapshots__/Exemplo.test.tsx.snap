// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Exemplo /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Exemplo
  </h1>
</main>
`;
